package.json

```diff
@@ -39,6 +39,7 @@
     "linkify-react": "^4.1.0",
     "linkifyjs": "^4.1.0",
     "lodash.truncate": "^4.4.2",
+    "match-sorter": "^6.3.1",
```

Comment:
Do you use match-sorter instead of fuse? Explain the motivation please

Reply:
There is a problem with fuse, where if you have title
`Polkadot Hispana`
and search `Polkahis`

it won't put Polkadot Hispana as result.
I have tried using their playground https://fusejs.io/demo.html but I couldn't find the config to do that
(with `minMatchCharLength` config set to search string length, because if not, it will show all match that matches at least only 2 characters)

match-sorter is also small (a bit smaller actually, fuse: 5kb, match-sorter: 3kb)